Python/other/minimize_the_jump.py

```diff
@@ -0,0 +1,40 @@
+#Find Minimum number of jumps to reach end
+def min_jump(arr, n):
+    if n <= 1: 
+        return 0 #return zero values if the array less than one element
+    
+    #assigning the value 0    
+    left = 0
+    right = arr[0]
+    count = 1
+
+    while right < n-1:
+        count += 1 #incresing the value 
+        jump = max(i + arr[i] for i in range(left, right + 1)) #by this checking all the arr and finding min values 
+        left, right = right, jump #swaping the values
+    return count
+
+arr = list(map(int, input("Enter the elements: ").split()))
+size = len(arr)
+print('Minimum number of jumps to reach',
+      'end is', min_jump(arr, size))
+
+'''
+Time Complexity : O(N)
```

Comment:
Please tell how the Time Complexity is **O(n)** in comment part

Reply:
By mistake, I wrote the O(n) it should be O(log(n)) actually bit hurry that why I wrote wrong time complexity